### PR TITLE
Fixed #33909 -- Corrected django.dispatch.receiver() signature.

### PR DESCRIPTION
**Pull request by badziyoussef**

### CODE REVIEW

File: docs/topics/signals.txt [hunk 1]
```diff
@@ -121,7 +121,7 @@ Here's how you connect with the decorator::
     from django.core.signals import request_finished
     from django.dispatch import receiver
 
-    @receiver(request_finished)
+    @receiver(request_finished, **kwargs)
```
Comment (felixxm) [inline docs/topics/signals.txt:124]: `kwargs` should be documented in the signature above:
```
.. function:: receiver(signal, **kwargs)
```
and in the list of params:
```
:param **kwargs: ...
```

Comment (badziyoussef) [reply docs/topics/signals.txt:125]: @felixxm the PR looks good now?

File: docs/topics/signals.txt [hunk 2]
```diff
@@ -112,16 +112,17 @@ manual connect route::
 
 Alternatively, you can use a :func:`receiver` decorator:
 
-.. function:: receiver(signal)
+.. function:: receiver(signal, **kwargs)
 
     :param signal: A signal or a list of signals to connect a function to.
+    :param kwargs: keyword arguments to connect a function to a signal or a list of signals.
```
Comment (felixxm) [inline docs/topics/signals.txt:118]: I added a reference to the `receiver-functions`.

**Review by felixxm — COMMENTED** [review]
@badziyoussef Thanks :+1:

**Comment by mlissner** [comment]
Nice. Thanks for jumping on this. I was planning on it, but also failing to act on those plans!

**Pull request merged**